### ISSUE TITLE
pact-jvm-consumer-junit: Decimal points are sometimes removed when writing to a pact consumer file

### ISSUE THREAD

**Issue by alexikin** [DiUS/pact-jvm#221]
Given I have a bundle with a post request body :
`.body("{\"responsetest\": true, \"name\": \"harry\",\"data\": 1234.0 }")`
When I run a consumer test with this body
Then I receive a response 
And I verify the response body
```
 assertEquals(,"harry",responseJSON.get("name"));
 assertEquals("Miss matched data",1234.0f,responseJSON.get("data"),0.1f);
```
And the pact file is written
And within the file there is a response
And EXPECTED within this response, there is a key value pair:   "data": 1234.0,
And ACTUAL within this response, there is a key value pair:   "data": 1234,

edit: formatting

**Comment by uglyog** [comment]
I'll need to investigate, but my guess would be that the JSON parser is dropping the zero.

**Comment by uglyog** [comment]
I've raised https://issues.apache.org/jira/browse/GROOVY-7794 to resolve this

**Comment by uglyog** [comment]
Turns out this has been fixed in Groovy 2.4.6, so I've upgraded it and it fixes your problem.

**Comment by alexikin** [comment]
Thanks for the very prompt response.
